test/extensions/one-tx-payment.js

```diff
@@ -37,14 +41,34 @@ contract("One transaction payments", accounts => {
     await colony.setAdministrationRole(1, 0, COLONY_ADMIN, 1, true);
     await colony.setFundingRole(1, 0, COLONY_ADMIN, 1, true);
 
-    oneTxExtension = await OneTxPayment.new(colony.address);
+    await oneTxExtensionFactory.deployExtension(colony.address);
+    const oneTxExtensionAddress = await oneTxExtensionFactory.deployedExtensions(colony.address);
+    oneTxExtension = await OneTxPayment.at(oneTxExtensionAddress);
```

Comment:
Can we add a test to check the `oneTxExtensionAddress` matches the new entry in Factory

Reply:
Same for `OldRolesFactory` tests as above

Reply:
I'm not really sure how you want me to do this that is different from what this already does - I would just be calling `oneTxExtensionFactory.deployedExtensions` again and checking it gave the same result?

Reply:
It'll be nice to check the contract created in `deployExtension` transaction (via the `contractAddress` property) matches `oneTxExtensionAddress`. Alternatively (as we do elsewhere in the network) we can trigger an event with that address which we check?

Reply:
I have ended up going the events route for consistency.